### ISSUE TITLE
Using square brackets to access properties only works one level

### ISSUE THREAD

**Issue by lieser** [jsdoc-type-pratt-parser/jsdoc-type-pratt-parser#142]
**Describe the issue**

Using square brackets to access properties currently only works one level. Using it more than once results in a parsing error.

**To Reproduce**

Input: `typeof obj["level1"]["level2"]`
Output by https://jsdoc-type-pratt-parser.github.io/jsdoc-type-pratt-parser/:
```
Error: The parsing ended early. The next token was: '[' with value '['
```

**Expected behavior**

Input should be accepted as a valid type.

**Additional context**

Note that the problem does not occur then using the dot notation.
Unlike jsdoc-type-pratt-parser, then mixing both notation, VS Code seems to not support the dot notation happening after using square brackets.

| Example | Works in VS Code | Works in jsdoc-type-pratt-parser |
|--------- |-------------------|------------------------------------|
| `typeof obj["level1"]["level2"]` | ✔ | ❌ |
| `typeof obj.level1.level2` | ✔ | ✔ |
| `typeof obj.level1["level2"]` | ✔ | ❌ |
| `typeof obj["level1"].level2` | ❌  | ✔ |

**Comment by simonseyock** [comment]
This is fixed and released in the `dev` branch right now.

**Comment by simonseyock** [comment]
This issue has been resolved in version 3.2.0-dev.3

**Comment by brettz9** [comment]
Looks like everything is working great, thanks! If you want to update to a regular release, I can do the same on our part.

**Comment by simonseyock** [comment]
It is included in v4.0.0
